### ISSUE TITLE
Support for Claude 3 models - Haiku, Sonnet, Opus

### ISSUE THREAD

**Issue by ayush-bharti** [zya/litellmjs#15]
Hi, thanks for working on this library.

Currently trying to use any of the Claude 3 models gives an error e.g. '"claude-3-haiku-20240307" is not supported on this API. Please use the Messages API instead.'

Are there any plans to support the new Claude 3 models?

**Comment by zya** [comment]
Yes, will be adding anthropic support soon. When I get a chance.
